apps/ejabberd/test/ejabberd_listener_SUITE_data/ejabberd.alt.cfg

```diff
@@ -0,0 +1,745 @@
+%%%
```

Comment:
I know we did it before for other ejabberd.cfg used in tests, but I think it'd be better to remove all the commented lines, for the sake of readability.